### ISSUE TITLE
Matching Specialist by name makes translation imposible

### ISSUE THREAD

**Issue by sharak** [drewg13/foundryvtt-band-of-blades#46]
Hi friend.

We are trying to translate to spanish your awesome system. 
We have almost everything working fine but we have a small problem in the Marshal's sheet.

On `role-sheet.file` (line 1225) the following appears:
![role-sheet.html](https://github.com/drewg13/foundryvtt-band-of-blades/assets/985713/74903a80-d97b-42ed-9918-117d33fb7764)

In our translation `specType.name` is, for instance, `Pesado` (Heavy on spanish) but `specialist.system.class` is `Heavy`. 

We have seen that there is a property in the item type "class" which is `class` and it is empty. Would it be possible to start using that property and record there the name of the class (untranslated) to make comparisons?

This would involve a single change in the code but you would have to fill this property in the compendium and perhaps enable it in the `class.html` to be able to update it.

That's the code change:
```
- {{#if (eq specialist.system.class specType.name)}}
+ {{#if (eq specialist.system.class specType.system.class)}}
```

Do you think this is possible?

Thank you!

**Comment by drewg13** [comment]
To a first pass, it seems perfectly reasonable to me.  Thanks for doing so much of the necessary investigation, I know that code is a hot mess.  I'll have a chance to test some code tomorrow, hopefully.

**Comment by drewg13** [comment]
Should be fixed in 2.4.0, just released

**Comment by sharak** [comment]
Thank you very much! It's working fine. 

I have detected another such error when trying to add a chosen/broken ability. Our popup shows empty.

The problem is in this line of the `bob-sheet.html` file.

![image](https://github.com/drewg13/foundryvtt-band-of-blades/assets/985713/f359f17c-e728-482c-9d2e-6e789285267d)

I think that the following can be added to the effect that the chosen/broken have[1]:
![image](https://github.com/drewg13/foundryvtt-band-of-blades/assets/985713/65c09a5e-1235-49f6-a073-a1d9f3a09b13)

And change line 110 on bob-sheet.js by:
```js
if ( e.system.class === this.actor.system.class )
```

[1] We tried to translate the "class" property of the chosenability objects but there are several items with the same name and Babele does not allow to translate several objects with the same name and different properties (look for the translation by the original name of the object)
https://gitlab.com/riccisi/foundryvtt-babele/-/issues/90
